README.md

```diff
@@ -73,17 +73,7 @@ If you find a problem, please [file a bug](https://github.com/mdn/browser-compat
 
 ## Contributing
 
-We're very happy to accept contributions to this data. Please familiarize yourself
-with the schema and send us a pull request. See also the [Contributing file](https://github.com/mdn/browser-compat-data/blob/master/CONTRIBUTING.md) for more information.
-
-## Browser compatibility tables on MDN
-
-It takes 1-2 weeks for changes in this data to be reflected in MDN's browser compatibility tables. The process is:
-
-1. A pull request is reviewed and merged to master.
-2. A new release of [mdn-browser-compat-data](https://www.npmjs.com/package/mdn-browser-compat-data) is created by MDN staff. This happens every 4-14 days.
-3. A new image of [Kumascript](https://github.com/mdn/kumascript), which includes the BCD release, is built and deployed to production. This happens within a day of the npm package release.
-4. The MDN page using the data is regenerated. For newly converted pages, a staff member switches to the [{{Compat}}](https://github.com/mdn/kumascript/blob/master/macros/Compat.ejs) macro, and re-checks the conversion. For updates to converted pages, a logged-in MDN user [force-refreshes the page](https://en.wikipedia.org/wiki/Wikipedia:Bypass_your_cache#Bypassing_cache) to regenerate it.
+We're very happy to accept contributions to this data. See the [Contributing to browser-compat-data](/docs/readme.md) for more information.
```

Comment:
"See the [Contributing to browser-compat-data] for more information" doesn't read like a complete sentence to me. 
Maybe "See the [Contributing to browser-compat-data] _docs_ for more information" ?

Reply:
Good catch. I think dropping "the" was the way to go here, but it's fixed now!